.ci/Jenkinsfile

```diff
@@ -343,6 +345,19 @@ pipeline {
             }
           }
         }
+        stage('Publish to npm') {
+          steps {
+            withNodeJSEnv(version: 'v14.17.5') {
```

Comment:
Nit: It might be a bit cleaner to set this as a variable at the top of the pipeline around L19 or so.

Reply:
done